### PR TITLE
18 mostly leaked DSK/DSC cards (plus one token)

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/the_lord_of_pain.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:The Lord of Pain
+ManaCost:3 B R
+Types:Legendary Creature Human Assassin
+PT:5/5
+K:Menace
+S:Mode$ CantGainLife | ValidPlayer$ Player.Opponent | Description$ Your opponents can't gain life.
+T:Mode$ SpellCast | ValidActivatingPlayer$ Player | ActivatorThisTurnCast$ EQ1 | Execute$ TrigDealDamage | TriggerZones$ Battlefield | TriggerDescription$ Whenever a player casts their first spell each turn, choose another target player. CARDNAME deals damage equal to that spell's mana value to the chosen player.
+SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Player | NumDmg$ X
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_lord_of_pain.txt:8]: try `Player.!TriggeredPlayer`

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/the_lord_of_pain.txt]: Should be !TriggeredActivator, I think, that's what's used in SpellCast triggers, right?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/the_lord_of_pain.txt]: yea probably better

File: forge-gui/res/cardsfolder/upcoming/barbflare_gremlin.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Barbflare Gremlin
+ManaCost:3 R
+Types:Creature - Gremlin
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/barbflare_gremlin.txt:3]: typo

File: forge-gui/res/cardsfolder/upcoming/convert_to_slime.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Convert to Slime
+ManaCost:3 B G
+Types:Sorcery
+A:SP$ Destroy | TargetMin$ 0 | TargetMax$ 1 | ValidTgts$ Creature | TgtPrompt$ ValidTgts$ Artifact | TgtPrompt$ Select target artifact to destroy | RememberDestroyed$ True | SubAbility$ DBDestroy1 | SpellDescription$ Destroy up to one target artifact, up to one target creature, and up to one target enchantment.\nDelirium — Then if there are four or more card types among cards in your graveyard, create an X/X green Ooze token, where X is the total mana value of permanents destroyed this way.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/convert_to_slime.txt:4]: Choose first and destroy together

File: forge-gui/res/cardsfolder/upcoming/dont_make_a_sound.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Name:Don't Make a Sound
+ManaCost:1 U
+Types:Instant
+A:SP$ Counter | TargetType$ Spell | TgtPrompt$ Select target creature or battle spell | ValidTgts$ Creature,Battle | UnlessCost$ 4 | UnlessResolveSubs$ WhenPaid | SubAbility$ DBSurveil | SpellDescription$ Counter target spell unless its controller pays {2}. If they do, surveil 2. (Look at the top two cards of your library, then put any number of them into your graveyard and the rest on top of your library in any order.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/dont_make_a_sound.txt:4]: wrong tgts?

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/dont_make_a_sound.txt:4]: 4?

File: forge-gui/res/cardsfolder/upcoming/gleeful_arsonist.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Gleeful Arsonist
+ManaCost:2 R
+Types:Creature - Human Wizard
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/gleeful_arsonist.txt:3]: typo

File: forge-gui/res/cardsfolder/upcoming/seance_board.txt [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Name:Séance Board
+ManaCost:2
+Types:Artifact
+T:Mode$ Phase | Phase$ End of Turn | CheckSVar$ X | SVarCompare$ GE1 | TriggerZones$ Battlefield | OptionalDecider$ You | Execute$ TrigPutCounter | TriggerDescription$ Morbid — At the beginning of each end step, if a creature died this turn, put a soul counter on CARDNAME.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/seance_board.txt:4]: not optional

File: forge-gui/res/cardsfolder/upcoming/shroudstomper.txt [hunk 7]
```diff
@@ -0,0 +1,12 @@
+Name:Shroudstomper
+ManaCost:3 W W B B
+Types:Creature Elemental
+PT:5/5
+K:Deathtouch
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigLoseLife | TriggerDescription$ Whenever CARDNAME enters or attacks, enters or attacks, each opponent loses 2 life. You gain 2 life and draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/shroudstomper.txt:6]: bad description

File: forge-gui/res/cardsfolder/upcoming/suspended_sentence.txt [hunk 8]
```diff
@@ -0,0 +1,7 @@
+Name:Suspended Sentence
+ManaCost:3 B
+Types:Instant
+A:SP$ Destroy | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | SubAbility$ DBLoseLife | SpellDescription$ Destroy target creature an opponent controls. That player loses 3 life. Exile Suspended Sentence with three time counters on it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/suspended_sentence.txt:4]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/convert_to_slime.txt
Comment (Fulgur14) [inline forge-gui/res/cardsfolder/upcoming/convert_to_slime.txt:1]: All Shall Smolder in My Wake will need a similar fix.

File: forge-gui/res/cardsfolder/upcoming/dont_make_a_sound.txt (hunk 4 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/dont_make_a_sound.txt]: still wrong

File: forge-gui/res/cardsfolder/upcoming/convert_to_slime.txt
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/convert_to_slime.txt:1]: indeed

**Review by tool4ever — DISMISSED** [review]

**Review by tool4ever — APPROVED** [review]

**Pull request merged**